### PR TITLE
GCloud GKE support via Terraform and Ansible

### PR DESCRIPTION
**Pull request by dougortiz**
1. Support for GCloud GKE via Terraform
2. CloudNativePG deployment on GKE
3. CloudNative Postgres deployment on GKE
4. Terraform Version: 1.2.6
5. Ansible Version:  2.12.18
6. GCloud CLI - Version: 398.0.0

### CODE REVIEW

File: edbdeploy/data/terraform/azure-aks/terraform.tfvars [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Get the appId and password with az cli command:
+# az ad sp create-for-rbac --skip-assignment
+appId            = "32d66de6-e8bd-41f6-8a58-327e3722dc7d"
+password         = "MYI.D1KxtQrdBV01k2x4L.Vp13nLfGWlyg"
```
Comment (bryan-bar) [inline edbdeploy/data/terraform/azure-aks/terraform.tfvars:4]: default password should be removed

File: edbdeploy/data/ansible/kubernetes_roles/remove_cloudnativepg_sandbox/README.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
+# remove_cloudnativepg_sandbox
```
Comment (bryan-bar) [inline edbdeploy/data/ansible/kubernetes_roles/remove_cloudnativepg_sandbox/README.md:1]: Duplicate role in edb-ansible:
https://github.com/EnterpriseDB/edb-ansible/tree/master/roles/remove_cloudnativepg_sandbox

File: edbdeploy/data/ansible/kubernetes_roles/remove_cloudnativepostgres_sandbox/README.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
+# remove_cloudnativepostgres_sandbox
```
Comment (bryan-bar) [inline edbdeploy/data/ansible/kubernetes_roles/remove_cloudnativepostgres_sandbox/README.md:1]: Duplicate role in edb-ansible:
https://github.com/EnterpriseDB/edb-ansible/tree/master/roles/remove_cloudnativepostgres_sandbox

File: edbdeploy/data/ansible/kubernetes_roles/setup_cloudnativepg_sandbox/README.md [hunk 4]
```diff
@@ -0,0 +1,91 @@
+# setup_cloudnativepg_sandbox
```
Comment (bryan-bar) [inline edbdeploy/data/ansible/kubernetes_roles/setup_cloudnativepg_sandbox/README.md:1]: Duplicate role in edb-ansible:
https://github.com/EnterpriseDB/edb-ansible/tree/master/roles/setup_cloudnativepg_sandbox

File: edbdeploy/data/ansible/kubernetes_roles/setup_cloudnativepostgres_sandbox/README.md [hunk 5]
```diff
@@ -0,0 +1,91 @@
+# setup_cloudnativepostgres_sandbox
```
Comment (bryan-bar) [inline edbdeploy/data/ansible/kubernetes_roles/setup_cloudnativepostgres_sandbox/README.md:1]: Duplicate role in edb-ansible:
https://github.com/EnterpriseDB/edb-ansible/tree/master/roles/setup_cloudnativepostgres_sandbox

**Review by bryan-bar — CHANGES_REQUESTED** [review]
Hello @dougortiz
please review the following:

- Duplicate roles available under `edb-ansible` should be used for `cloudnativepostgres` and `cloudnativepg`
- 2 role directories with the same tasks: `edbdeploy/ansible/roles` and `edbdeploy/ansible/kubernetes-roles`. Are both needed?
- default `appid` and `password` is currently set for azure

**Review by bryan-bar — CHANGES_REQUESTED** [review]
Merge conflict in `edbdeploy/ansible.py`

**Comment by dougortiz** [comment]
Request another review

**Comment by dougortiz** [comment]
GKE refactoring of code

**Comment by dougortiz** [comment]
GKE was refactored via PR: 335

**Comment by dougortiz** [comment]
Squash and merged

**Pull request merged**